.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = doc/examples
 	url = https://github.com/mozilla/DeepSpeech-examples.git
 	branch = master
+[submodule "tensorflow"]
+	path = tensorflow
+	url = https://github.com/mozilla/tensorflow.git
+	branch = r2.2
```

Comment:
I think we shouldn't use the automatic branch tracking here, because there is a dependency between updating the tensorflow reference and invalidating the CI build caches that will be broken if you just update tensorflow unilaterally. This submodule should point to a fixed commit.